include/rocksdb/metadata.h

```diff
@@ -146,6 +149,43 @@ struct LiveFileMetaData : SstFileMetaData {
   LiveFileMetaData() : column_family_name(), level(0) {}
 };
 
+// The MetaData that describes a Blob file
+struct BlobMetaData {
```

Comment:
Please add the blob file's size as well (can be computed as `total_blob_bytes` + size of blob file header + size of blob file footer).